aries-site/src/examples/templates/SidebarExample.js

```diff
@@ -0,0 +1,61 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Button, ResponsiveContext } from 'grommet';
+import {
+  Apps,
+  Chat,
+  Clock,
+  Configure,
+  Terminal,
+  ShieldSecurity,
+  StatusUnknown,
+} from 'grommet-icons';
+
+export const SidebarExample = ({ direction, ...rest }) => {
```

Comment:
Thoughts on naming this `GlobalNavBar` vs. `Sidebar` to align terminology with: https://github.com/hpe-design/design-system/issues/349?

`GlobalNavBar` may be more explicit so that when we see it, we know that it refers to the blue bar for navigation between HPE apps/services and is sometimes a sidebar, but also a bar at the footer on small screens.

Reply:
I am open to this naming -- I'm not seeing `Global` on that link however I think it does align with previous conversations of `GlobalLayout` which contains this NavBar and `AppLayout`. Thinking this naming convention (and potentially a component) would be handled by the work Shimi mentioned she'd want to explore in the development of those components. Not sure if we want to resolve naming now or let that development work happen and then we can adjust this example afterwards.